src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -141,7 +177,7 @@ const useGroupSetting = ({navigation, route}) => {
     isLoading,
     setIsLoading,
     handleResLaunchGallery,
-    updateDataEdit
+    loadingUpdate
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `loadingUpdate` state variable is exposed but it's not clear how it's used outside of this hook. If it's not used elsewhere, consider removing it from the returned object.

```diff
- loadingUpdate
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->